Jenkinsfile

```diff
@@ -35,6 +39,19 @@ pipeline {
 
     stages {
 
+        stage('Cloning Repos') {
+            steps {
+                withEnv(["WORK_HOME=${env.WORKSPACE}"]) {
+                    sh """
+                        git clone https://github.com/lsst-ts/ts_m2com.git
+
+                        cd ${WORK_HOME}/ts_m2com
+                        git checkout -t origin/${env.BRANCH} | true
```

Comment:
This might fail if there is no such branch in `ts_m2com`:

```text
(base) teweitsai@te-weis-mbp-2:test$ git clone --branch abc https://github.com/lsst-ts/ts_m2com.git
Cloning into 'ts_m2com'...
fatal: Remote branch abc not found in upstream origin
```

Thanks!